### PR TITLE
empty MANIFEST.in seems to be doing the right thing

### PR DESCRIPTION
**Pull request by dholth**
### Description

Simplify MANIFEST.in which was pointing at non-existent files.

### Checklist - did you ...

- [x] Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?
     
- [x] Add / update necessary tests?
- [x] Add / update outdated documentation?

### CODE REVIEW

**Comment by jezdez** [comment]
Shouldn't we simply migrate to `pyproject.toml` instead?

File: MANIFEST.in
Comment (jezdez) [inline MANIFEST.in:1]: Might as well delete the file, right?

Comment (dholth) [reply MANIFEST.in:1]: I've developed entire setuptools alternatives to get rid of this file

Comment (dholth) [reply MANIFEST.in:1]: Removed.

**Review by jezdez — APPROVED** [review]

**Pull request merged**